### PR TITLE
Add product Poof Links

### PR DESCRIPTION
**Pull request by sjoseph7**
**Type of pull request:** product addition 

*Full Disclosure: I am the creator of Poof Links.*

### CODE REVIEW

File: products/pooflinks.toml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+[rubric.data-collection-reasoning]
+value = "yes"
+citations = ["\"Unsurprisingly, we collect the secret messages you submit to us. While we don't use cookies, we do log IP addresses to monitor usage and prevent abuse.\""]
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["\"A secret message can only be retrieved within the window of time specified by the submitting party, up to a maximum of 7 days, and secrets that have not been retrieved within the designated window are considered 'expired'. Secrets are wiped from our servers once viewed, and secrets that expire are immediately inaccessible and are permanently removed from our database within two (2) minutes of their expiration.\""]
+
+[rubric.history]
```
Comment (doamatto) [inline products/pooflinks.toml:25]: Not required, but it’s nice to have the date provided here as a mini timestamp to see if it needs updating (while we wait on me to finish the scrubber)

File: products/pooflinks.toml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+[rubric.revision-notify]
+value = "na"
+notes = ["The service only collects IP addresses, so notification would be impossible."]
+
+[rubric.security]
+value = "yes"
+citations = [
+  "\"When we receive a message submission, we immediately secure it with AES-256 (a.k.a. 'military-grade') encryption using a randomly generated symmetric key and initialization vector for each message. Your secret message is never written into our server logs, and we never write unencrypted messages to our databases. We embed the key in the provided url and don't store keys in our databases for www.pooflinks.com and any of our browser extensions.\"",
+  "\"If you lose your URL, we cannot retrieve your secret for you.\""
```
Comment (doamatto) [inline products/pooflinks.toml:49]: Doesn't really pertain to data protection imo

**Review by doamatto — CHANGES_REQUESTED** [review]
Overall solid; a few tweaks I would make though

File: products/pooflinks.toml (hunk 2 above)
Comment (sjoseph7) [reply products/pooflinks.toml:49]: Do you have any recommendations for improving this section?

Comment (milesmcc) [reply products/pooflinks.toml:50]: In general I think it's good to avoid references to "military grade" encryption.

Comment (sjoseph7) [reply products/pooflinks.toml]: I've modified the notes/citations and downgraded the rating.

**Review by doamatto — APPROVED** [review]
Looks good now; cheers

**Pull request merged**